docs/getting-started-locally.md

```diff
@@ -0,0 +1,93 @@
+# Starting an Interledger Node Locally Using moneyd
+
+This tutorial describes how to:
+1. Install [`moneyd`](../concepts/moneyd.md) on your system
+2. Start an [Interledger](https://github.com/interledgerjs/ilp-connector) node on a local test network using moneyd
+3. Send and receive payments using the [SPSP](../concepts/spsp.md) (Simple Payment Setup Protocol) API
+
+This tutorial is useful if you’re doing local development. 
+
+![Getting-Started-Locally](../assets/getting-started-locally.png)
+
+## Before you begin
+* Install a stable version of [Node.js](https://nodejs.org/en/) (10.16.0 LTS is recommended)
+* Set up Node to install modules globally <br/>
+Open a command line and use the following commands:  
+  * `mkdir <new-npm-dir>`  
+  * `npm config set prefix <path-to-new-npm-dir>`  
+  * Add `<new-npm-dir>` to your system’s `PATH` environment variable
+
+NOTE: For this tutorial you *do not* need to use any cryptocurrency.
+
+## Installing moneyd
+To install `moneyd`, open a command line and run the following command:
+```shell
+npm install -g moneyd
+```
+
+## Starting a node
+After you’ve installed `moneyd`, run the following command to start your local node:
+```shell
+moneyd local
+```
+Running the above command creates an Interledger node that listens on port 7768.
+
+## Sending and receiving payments
+Once you have `moneyd` running, you can send and receive payments over ILP using the SPSP API. For this tutorial, we’ll use the SPSP command line tool.
+
+### Installing the SPSP client and server
+To install an SPSP client and server, open a new command line and run:
+```shell
+npm install -g ilp-spsp ilp-spsp-server
+```
+### Starting the SPSP server
+To receive payments, start the SPSP server using the following command:
+```shell
+ilp-spsp-server --subdomain my_subdomain
+```
+Replace `my_subdomain` with the name of your choice or a random string. 
+
+### Sending a payment
+Now, to send a payment, open another terminal and run:
+
+*For Mac and Linux users*:
+```shell
+ilp-spsp send --amount 10 --receiver '$my_subdomain.localtunnel.me'
```

Comment:
if you do the suggestion above of not using localtunnel, the string will be the same between linux/osx/windows